argoverse/data_loading/object_label_record.py

```diff
@@ -207,6 +238,56 @@ def draw_rect(selected_corners: np.array, color: Tuple[int, int, int]) -> None:
         return img
 
 
+def add_text_cv2(img: np.ndarray, text: str, x: int, y: int, color: Tuple[int,int,int], thickness: int = 3) -> None:
+    """Add text to image using OpenCV. Color should be RGB order"""
+    img = cv2.putText(
+        img,
+        text,
+        (x,y),
+        fontFace=cv2.FONT_HERSHEY_SIMPLEX,
+        fontScale=1,
+        thickness=thickness,
+        color=color
+    )
+
+
+def draw_alpha_rectangle(
+    img: np.ndarray,
+    top_left: Tuple[int,int],
+    bottom_right: Tuple[int,int],
+    color_rgb: Tuple[int,int,int],
+    alpha: float
+):
+    """Alpha blend colored rectangle into image. Coords given as (x,y) tuples"""
```

Comment:
Should this have args in the docstring?

Reply:
not sure, i think is self explanatory? let me know if not though